kernel/Kconfig

```diff
@@ -713,6 +714,7 @@ config EVENTS
 
 config PIPES
 	bool "Pipe objects"
+	select DEPRECATED
```

Comment:
This might be seen as a deprecation of pipes, what you are doing here is deprecating the kconfig only, worth some text in the help section.